lib/tasks/asset_sync.rake

```diff
@@ -10,21 +9,4 @@ namespace :assets do
       AssetSync.clean
     end
   end
-
-end
-
-if Rake::Task.task_defined?("assets:precompile:nondigest")
```

Comment:
Hm, now that I look at it, I might want to leave that in -- at least the code on line 28. Otherwise, I'm not entirely sure how or where we enhance the existing `assets:precompile` task. I wonder if the author of Asset Sync PR 345 expected us to run an explicit `rake asset:sync` task instead of calling an enhanced `rake assets:precompile`.

Perhaps we might want the code from https://github.com/AssetSync/asset_sync/pull/341/files#diff-0237b97660767b03106c26b0115c3621R23

??